TKOaly/rv-app-frontend
# Add feature to filter products based on their stock

PR checklist



* [x] Give a description about your pull request.

------
* [x] Create testing instructions for your PR.

------

* [x] Add tags to your PR. 

Add tags to your PR to indicate the status of your PR. `WIP` for work in progress, `Ready for review` if your PR is ready for review. You can view all the tags from the right side panel.

------

* [x] Assign a code reviewer to your PR.

Assign one or more reviewers to review your code changes. Merging a PR requires at least one code reviewer to give an approval.

------

* [ ] Notify your team about merging the PR if you think its wise.

## Description

This PR adds functionality to filter products based on their current stock. You can add products to a whitelist, so coffee and bottle/can returns do not get hidden when their stock goes below 1.

## Testing instructions

Try to buy items so that their stock goes negative. Verify that the item disappears from the UI.

## Other

-
